File: WebUI/AkademiaApplication/src/app/models/application.ts
import { ApplicationStatus } from "./applicationStatus"

export interface Application
{
    id: number | null
    number: string | null
    title: string | null
    description: string | null
    applicantsName: string | null
    createdDate: Date | null
    applicationStatus: ApplicationStatus | null
    completionDate: Date | null
}

File: Services/Application.API/Domain/Application.cs
namespace Application.API.Domain
{
    public class Application
    {
        public int Id { get; set; }
        public string Number { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string ApplicantsName { get; set; }
        public DateTime CreatedDate { get; set; }
        public ApplicationStatus ApplicationStatus { get; set; }
        public DateTime CompletionDate { get; set; }
    }
}
